doc/jsk_pcl_ros/nodes/attention_clipper.md

```diff
@@ -8,6 +8,14 @@ retrieves `sensor_msgs/PointCloud2` and publish `pcl_msgs/PointIndices`.
 You can specify the pose and size of the interest bounding box and jsk\_pcl/AttentionClipper returns ROI
 to see the object.
 
+## Note
+
+AttentionClipper does not work properly on Kinetic + PCL 1.8.0, and you can avoid the problem with SSE disabled PCL.
```

Comment:
Also on melodic?

Reply:
I didn't check it on melodic.
Did you check it on melodic?

Reply:
Oops, sorry, I was confused with kinfu.
Please forget my comment.